dev-docs/bidders/colombia.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: bidder
+title: Colombia
+description: Prebid Colombia Bidder Adaptor
+biddercode: colombia
+media_types: banner, video, native
+usp_supported: true
```

Comment:
Sorry, I can't approve declaring that you support US Privacy when all you're doing is forcing the publisher to set a boolean flag. Unclear what that means.

You have two choices:

1. set this flag value to `false`
2. update your adapter to read the us_privacy string from the bidrequest object and send it to your endpoint. Many other adapters do this.

Reply:
Removed